src/main/java/uk/gov/ons/census/fwmt/outcomeservice/controller/OutcomeApi.java

```diff
@@ -46,4 +47,13 @@
       method = RequestMethod.POST)
   ResponseEntity<CCSInterviewOutcome> ccsInterviewOutcome(
       @PathVariable String caseId, @RequestBody CCSInterviewOutcome ccsInterviewOutcome) throws GatewayException;
+
+  @ApiOperation(value = "Post a SPG survey outcome to the FWMT Gateway", response = HouseholdOutcome.class)
```

Comment:
Youve got HouseHoldouldPOutcome as a return type. No object is to be return, i.e. Void